vm/class_test.go

```diff
@@ -990,3 +990,24 @@ func TestClassSingletonClassClassMethod(t *testing.T) {
 		v.checkSP(t, i, 1)
 	}
 }
+
+func TestObjectId(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected interface{}
+	}{
+		{`a = nil.object_id; b = nil.object_id; a == b`, true},
+		{`a = true.object_id; b = true.object_id; a == b`, true},
+		{`a = false.object_id; b = false.object_id; a == b`, true},
+		{`a = 1.object_id; b = 1.object_id; a == b`, false},
+		{`a = "a".object_id; b = "a".object_id; a == b`, false},
```

Comment:
I looked ruby/spec's `#object_id` spec, we should add test for the `@return [Integer]`
https://github.com/ruby/spec/blob/e84c928a9ce171693a1221da88cf98e3837a77fa/shared/kernel/object_id.rb
```rb
{`Object.new.object_id.is_a?(Integer)`, true}
```

Reply:
Added